src/containers/Layout/Layout.tsx

```diff
@@ -169,7 +169,9 @@ const Layout = () => {
           <hr className={styles.divider} />
           {renderUserActions()}
         </Sidebar>
-        <Outlet />
+        <div id="content">
+          <Outlet />
+        </div>
```

Comment:
I understand this decision, but adding an additional div could add issues, such as that `max-height` or `min-height` not working correctly. We should test it well, also on mobile devices perhaps, but it would be safer to add the id to all screens (pages) in stead.

Reply:
Good point!

Reply:
I needed to add a `<div>` on the Home page as well, so that would also add a new DOM node, just like I’m currently doing now in Layout. I therefore proceeded with the solution provided by Mike and added `tabIndex={-1}`.